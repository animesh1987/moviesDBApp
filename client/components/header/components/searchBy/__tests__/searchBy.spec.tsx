import * as React from 'react';
import { create } from 'react-test-renderer';
/*import { shallow } from 'enzyme';*/

import { SearchBy } from '../index';

describe('Search By test', () => {

  const toggleType = jest.fn();

  it('Search By renders correctly', () => {
    const component = create(
      <SearchBy
        containerClass={'test'}
        genreSelected={'title'}
        toggleType={toggleType}
      />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });

  /*it('Search by events work', () => {
    const component = shallow(
      <SearchBy
        containerClass={'test'}
        genreSelected={'title'}
        toggleType={toggleType}
      />
    );
    console.log(component.find('Button'));
    component.find('span')//.simulate('click');
    expect(toggleType).toBeCalled();
  });*/
});
