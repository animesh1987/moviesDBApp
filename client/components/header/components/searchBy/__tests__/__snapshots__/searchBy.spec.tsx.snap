// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search By test Search By renders correctly 1`] = `
<div
  className="test"
>
  <span
    className="search-by"
  >
    Search By
  </span>
  <button
    className="active"
    onClick={[Function]}
  >
    Title
  </button>
  <button
    className=""
    onClick={[Function]}
  >
    Genre
  </button>
</div>
`;
